src/commands/trialApp/cloneTrialApp.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { commands } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { TrialAppTreeItem } from '../../explorer/trialApp/TrialAppTreeItem';
+import { ext } from '../../extensionVariables';
+
+export async function cloneTrialApp(context: IActionContext, node?: TrialAppTreeItem): Promise<void> {
+    if (!node) {
```

Comment:
So you only need this if you enter through the command palette, which you've disabled.  I think you can just remove the ? in the parameter.